.circleci/config.yml

```
@@ -499,6 +524,17 @@ jobs:
           paths:
             - changed-files
 
+  validate-locales-only:
+    executor: node-browsers-small
+    steps:
+      - run: *shallow-git-clone
+      - run: sudo corepack enable
+      - attach_workspace:
+          at: .
+      - run:
+          name:
+          command: npx tsx .circleci/scripts/validate-locales-only.ts
```

Use `yarn` here instead to ensure that we aren't accidentally re-installing it here. `yarn tsx` will fail if it's not installed, which is what we want. Installing it at runtime means the lock file won't be used, so we're more susceptible to supply-chain attacks and inconsistencies between runs.
```suggestion
          command: yarn tsx .circleci/scripts/validate-locales-only.ts
```